📝 add more troubleshooting docs (#803)
mondoohq/mondoo-operator: docs/user-manual.md

    enable: true
```

### Why are (some of) my nodes unscored?

> In some cases the node scanning may require more memory than what we have initially allowed. You can check whether that is the case by running:

Reviewer: ```suggestion
In some cases a node scan can require more memory than initially allotted. You can check whether that is the case by running:
```

```bash
kubectl get pods -n mondoo-operator
```

> Look for pods in the form of <mondooauditconfig-name>-node-<node-name>-hash. For example, if your `MondooAuditConfig` is called `mondoo-client` and you have a node called `node01`, you should be able to find a pod `mondoo-client-node-node01-<hash>`. If the pods is crashing and restarting it is most probably running out of memory and getting killed. You can verify that by looking into the pod's status `kubectl get pods -n mondoo-operator mondoo-client-node-node01-<hash> -o yaml`.

Reviewer: ```suggestion
Look for pods in the form of `<mondooauditconfig-name>-node-<node-name>-hash`. For example, if your `MondooAuditConfig` is called `mondoo-client` and you have a node called `node01`, you should be able to find a pod `mondoo-client-node-node01-<hash>`. 

If the pod is crashing and restarting, it's most probably running out of memory and terminating. You can verify that by looking into the pod's status: 

```bash
kubectl get pods -n mondoo-operator mondoo-client-node-node01-<hash> -o yaml
```
```

GH won't let me do this in the suggestion  😄 : I think the last line here should be a code block.

> In case you need to increase the resource limits for node scanning you can alter your `MondooAuditConfig`:

Reviewer: ```suggestion
If you need to increase the resource limits for node scanning, change your `MondooAuditConfig`:
```

```bash
kubectl edit -n mondoo-operator mondooauditconfig mondoo-client
```

> Search for the `nodes:` section specify the new limits there. It should look like this:

Reviewer: ```suggestion
Search for the `nodes:` section and specify the new limits there. It should look like this:
```

```yaml
spec:
  nodes:
    enable: true
    resources:
      limits:
        cpu: 200m
        memory: 200Mi
```

### How can I trigger a new scan?

> The operator will run a full cluster scan and node scans hourly. In case you need to manually trigger those scans there are 2 options:

Reviewer: ```suggestion
The operator runs a full cluster scan and node scans hourly. If you need to manually trigger those scans there are three options:
```

> 1. Create a job from the existing cron job

Reviewer: ```suggestion
Option A: Create a job from the existing cron job
```

>     1. Locate the cronjob you want to trigger:

Reviewer: ```suggestion
    1. Locate the cron job you want to trigger:
```

    ```bash
    kubectl get cronjobs -n mondoo-operator
    ```

>     2. Create a new job from the existing cronjob. If we want to trigger a new cluster scan, the command will be:

Reviewer: ```suggestion
    2. Create a new job from the existing cron job. To trigger a new cluster scan, the command is:
```

    ```bash
    kubectl create job -n mondoo-operator my-job --from=cronjob/mondoo-client-k8s-scan
    ```

>     3. The scan will be started immediately by a job called `my-job`

Reviewer: ```suggestion
    3. A job called `my-job` starts the scan immediately.
```

> 2. Turn scanning off and then on again

Reviewer: ```suggestion
Option B: Turn scanning off and then on again
```

>     1. Edit the `MondooAuditConfig`

Reviewer: ```suggestion
    1. Edit the `MondooAuditConfig`:
```

    ```bash
    kubectl edit -n mondoo-operator mondooauditconfig mondoo-client
    ```

>     2. Disable scanning by changing `enable: true` to `enable: false`

Reviewer: ```suggestion
    2. Disable scanning by changing `enable: true` to `enable: false`:
```

    ```yaml
    spec:
      kubernetesResources:
        enable: false
      nodes:
        enable: false
    ```

>     3. Make sure the scan cronjobs are deleted before proceeding:

Reviewer: ```suggestion
    3. Make sure the scan cron jobs are deleted before proceeding:
```

    ```bash
    kubectl get cronjobs -n mondoo-operator
    ```

>     4. Edit the `MondooAuditConfig` again and turn re-enable scanning:

Reviewer: ```suggestion
    4. Edit the `MondooAuditConfig` again and re-enable scanning:
```

    ```yaml
    spec:
      kubernetesResources:
        enable: true
      nodes:
        enable: true
    ```

>     5. The scan cronjobs will be re-created and their initial run will be within the next 1 minute

Reviewer: ```suggestion
    5. The scan cron jobs will be re-created and their initial run will occur within the next minute.
```
